tests/integration/poddefault_test_workloads.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: user
```

Comment:
it probably wont matter, but maybe we should name this namespace something unlikely to collide with anything else.  Like `admission-webhook-test-user-namespace` or something similarly odd

Reply:
fixed .. good catch